src/components/atoms/Icon/index.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+
+import * as svg from './svg'
+
+export const iconTypes: IconType[] = Object.keys(svg) as any[]
+
+export type IconType = keyof typeof svg
+
+export type IconProps = {
+  name: IconType
+  className?: string
+  style?: React.CSSProperties
+  onClick?(): void
+}
```

Comment:
이거 저번에 https://github.com/Lubycon/clelab-frontend/pull/3 리뷰 코멘트에 달렸던 내용인 것 같아서 끌올해봅니다! 

```suggestion
export interface IconProps {
  name: IconType
  className?: string
  style?: React.CSSProperties
  onClick?(): void
}
```

Reply:
https://github.com/Lubycon/clelab-frontend/pull/7/commits/aa5229c9ec56f2ca9568c0009e6c6d40661abec3 에서 수정하였습니다. 
저번에 만들어 놓은 code snippet 쓰니까 자꾸 같은 실수를 반복 하네요 😅